nipype/workflows/rsfmri/fsl/resting.py

```diff
@@ -3,7 +3,6 @@
 # vi: set ft=python sts=4 ts=4 sw=4 et:
 from __future__ import (print_function, division, unicode_literals,
                         absolute_import)
-from builtins import str
```

Comment:
Why?

Reply:
That was an unnecessary import, I think.